java/opendcs/src/main/java/decodes/hdb/algo/EOPInterpAlg.java

```diff
@@ -30,22 +27,18 @@
 import decodes.hdb.dbutils.DataObject;
 import decodes.tsdb.DbCompException;
 import decodes.hdb.dbutils.RBASEUtils;
-//AW:IMPORTS_END
+import org.opendcs.annotations.PropertySpec;
+import org.opendcs.annotations.algorithm.Algorithm;
+import org.opendcs.annotations.algorithm.Input;
+import org.opendcs.annotations.algorithm.Output;
 
-//AW:JAVADOC
-/**
-Type a javadoc-style comment describing the algorithm class.
- */
-//AW:JAVADOC_END
+@Algorithm(description = "Type a javadoc-style comment describing the algorithm class.")
```

Comment:
Looking at the algorithm I would suggest something like

'interpolate End-Of-Period value'

instead of the default javadoc template for the description.